firebase.json

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
Did you want to check this file in or could it be autogenerated? Since we clobber the redirects every time and all.

Reply:
yeah i think we should check this file in and remove redirects.yml